File: whdon_app/src/main/java/nl/whitedove/washetdroogofniet/StatistiekWind.java
package nl.whitedove.washetdroogofniet;

class StatistiekWind {

    private WeerHelper.WindDirection WindDir;
    private String windOmschrijving;
    private int aantal;
    private float avgWindSpeed;
    private float maxWindSpeed;
    private float percentage;

    String getWindOmschrijving() {
        return windOmschrijving;
    }

    void setWindOmschrijving(String windOmschrijving) {
        this.windOmschrijving = windOmschrijving;
    }

    int getAantal() {
        return aantal;
    }

    void setAantal(int aantal) {
        this.aantal = aantal;
    }

    WeerHelper.WindDirection getWindDir() {
        return WindDir;
    }

    void setWindDir(WeerHelper.WindDirection windDir) {
        WindDir = windDir;
    }

    float getPercentage() {
        return percentage;
    }

    void setPercentage(float percentage) {
        this.percentage = percentage;
    }

    float getMaxWindSpeed() {
        return maxWindSpeed;
    }

    void setMaxWindSpeed(float maxWindSpeed) {
        this.maxWindSpeed = maxWindSpeed;
    }

    float getAvgWindSpeed() {
        return avgWindSpeed;
    }

    void setAvgWindSpeed(float avgWindSpeed) {
        this.avgWindSpeed = avgWindSpeed;
    }
}

File: whdon_app/src/main/java/nl/whitedove/washetdroogofniet/StatistiekWind.kt
package nl.whitedove.washetdroogofniet

class StatistiekWind {

    var windDir: WeerHelper.WindDirection = WeerHelper.WindDirection.Onbekend
    var windOmschrijving: String = ""
    var aantal: Int = 0
    var avgWindSpeed: Float = 0.toFloat()
    var minWindSpeed: Float = 0.toFloat()
    var maxWindSpeed: Float = 0.toFloat()
    var percentage: Float = 0.toFloat()
}
